bfv/bfv_test.go

```diff
@@ -227,413 +228,607 @@ func testEncoder(testctx *testContext, t *testing.T) {
 				coeffsInt[i] = int64(c)
 			}
 		}
-		plaintext := NewPlaintext(testctx.params)
-		testctx.encoder.EncodeInt(coeffsInt, plaintext)
-		coeffsTest := testctx.encoder.DecodeIntNew(plaintext)
+		plaintext := NewPlaintext(tc.params)
+		tc.encoder.EncodeInt(coeffsInt, plaintext)
+		coeffsTest := tc.encoder.DecodeIntNew(plaintext)
 
 		require.True(t, utils.EqualSliceInt64(coeffsInt, coeffsTest))
 	})
 
-	t.Run(testString("Encoder/Encode&Decode/PlaintextMul", testctx.params), func(t *testing.T) {
-		values, plaintext := newTestVectorsMul(testctx, t)
-		verifyTestVectors(testctx, nil, values, plaintext, t)
+	t.Run(testString("Encoder/Encode&Decode/PlaintextMul", tc.params), func(t *testing.T) {
+		values, plaintext := newTestVectorsMul(tc, t)
+		verifyTestVectors(tc, nil, values, plaintext, t)
 	})
 }
 
-func testEvaluator(testctx *testContext, t *testing.T) {
+func testQuantize(tc *testContext, t *testing.T) {
```

Comment:
should be part of the `testEvaluator` function since the quantize function is part of the `Evaluator` interface.

Reply:
Agree

Reply:
Done